integration-tests/contracts/contract_deployer.go

```diff
@@ -1307,6 +1394,22 @@ func (e *EthereumContractDeployer) LoadKeeperRegistry(address common.Address, re
 			registry2_1: instance.(*iregistry21.IKeeperRegistryMaster),
 			version:     registryVersion,
 		}, err
+	case eth_contracts.RegistryVersion_2_2: // why the contract name is not the same as the actual contract name?
```

Comment:
@anirudhwarrier do you know why?

Reply:
No particular reason. The name here doesn't matter, only the name in the contract will show up on chain when deployed. Feel free to change the name here to keep it in sync with the contract name.